.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "mainnet-contracts/lib/interfaces"]
```

Comment:
We've moved away from using submodules. Instead you can do a yarn install of this.